lldb/test/API/lang/swift/reflection_only/lib.swift

```diff
@@ -0,0 +1,34 @@
+public struct S {
+  public init() {}
+  public let pub = 1
+  private let priv = 2
+  fileprivate let filepriv = 3
+
+  private let s_priv = SPriv()
+  fileprivate let s_filepriv = SFilePriv()
+}
+
+private struct SPriv {
+  let i = 2
+}
+
+fileprivate struct SFilePriv {
+  let i = 3
+}
+
+open class Base {
+  public let x : Int = 42
+  public init() {}
+}
+
+public enum NoPayload {
+  case first
+  case second
+}
+
+public enum WithPayload {
+  case empty
+  case with(i: Int)
+}
```

Comment:
Just a thought: should we include a multi-payload enum here, commented out, with a link to the radar?